MAINTAINERS.yml

```diff
@@ -458,22 +458,23 @@ Display drivers:
   labels:
     - "area: Display"
 
-Documentation:
+Documentation Infrastructure:
```

Comment:
We should not be removing the `Documentation` entry altogether. If the area currently lacks a maintainer, then there needs to be an entry for it with `status: odd fixes`.

My suggestion would be to keep the current `Documentation` entry, but to add `files-exclude:` for the "infrastructure" portion under the `doc/` directory, and add a separate `Documentation Infrastructure` entry.

There will be some duplicate information between the `files-exclude:` for the `Documentation` entry and the `files:` for the `Documentation Infrastructure` entry, but this is unavoidable at this time (could be improved in the future as part of the ongoing MAINTAINERS file format rework).

Reply:
I've changed strategy, can you guys check if latest patch sounds reasonable?